spec/controllers/conference_registration_controller_spec.rb

```diff
@@ -0,0 +1,255 @@
+require 'spec_helper'
+
+describe ConferenceRegistrationsController, type: :controller do
+  let(:conference) { create(:conference) }
+  let(:conference_with_open_registration) { create(:conference, registration_period: open_registration_period) }
+  let(:open_registration_period) { create(:registration_period, start_date: Date.current - 6.days) }
```

Comment:
> Can you please define conference before you define the registration_period? It would be more clear what you are doing, because the logic is: I create a conference, then I create a registration_period for this conference. 
The same applies for other tests.
@differentreality 

[outdated diff](https://github.com/openSUSE/osem/pull/795#discussion_r54487416) :cry: